shared/src/native-src/pal.h

```diff
@@ -159,16 +173,18 @@ inline WSTRING GetCurrentProcessCommandLine()
     std::string name;
     std::stringstream tokens(cmdline);
     std::string tmp;
+    int i = 0;
     while (getline(tokens, tmp, '\0'))
     {
         name = name + " " + tmp;
+        args.push_back(Trim(ToWSTRING(tmp)));
     }
-    fclose(fp);
```

Comment:
why removing the `close` statement ? Since it was opened on line 164, it should be closed. You can move it up I think

Reply:
Yeah, that was definitely unintentional thanks 😄

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/5564/commits/594e711959b590150fd6f1f1aeb2f7c222b239b8